src/super_gradients/training/models/segmentation_models/ddrnet.py

```diff
@@ -369,15 +372,15 @@ def __init__(
                 strides=spp_strides,
             )
 
-            if self.aux_head:
-                self.seghead_extra = SegmentHead(highres_planes, head_width, num_classes, 8, inter_mode=self.segmentation_inter_mode)
-
             self.final_layer = SegmentHead(highres_planes * layer5_bottleneck_expansion, head_width, num_classes, 8, inter_mode=self.segmentation_inter_mode)
 
+            if self.use_aux_heads:
```

Comment:
Changing the order allows loading pretrained checkpoints with auxiliary heads, to a model without auxiliary heads